src/Symfony/Component/Console/Command/Command.php

```diff
@@ -429,9 +429,9 @@ public function addArgument(string $name, int $mode = null, string $description
     /**
      * Adds an option.
      *
-     * @param $shortcut The shortcuts, can be null, a string of shortcuts delimited by | or an array of shortcuts
-     * @param $mode     The option mode: One of the InputOption::VALUE_* constants
-     * @param $default  The default value (must be null for InputOption::VALUE_NONE)
+     * @param string|array|null         $shortcut The shortcuts, can be null, a string of shortcuts delimited by | or an array of shortcuts
+     * @param int|null                  $mode     The option mode: One of the InputOption::VALUE_* constants
```

Comment:
our description is about wrong here. It is a bitmask of those constants.

Reply:
Well the `InputOption->__construct` docblock is the same / more different that is listed here.
https://github.com/symfony/symfony/blob/137aa3ba3c73e823e8259352beaae66d25f13def/src/Symfony/Component/Console/Input/InputOption.php#L55-L62